### PR TITLE
Add first round of unit tests for topics retrieval

### PR DESCRIPTION
**Pull request by alexcn3**

### CODE REVIEW

File: project/src/test/java/com/google/sps/TrendRSSFeedTest.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    TrendRSSFeed testfeed = parser.readTrendsFromStream(new FileInputStream(new File("/home/acnwigwe/step19-2020/project/src/test/java/com/google/sps/RSSFeed.xml")));
+    Assert.assertNotNull(testfeed);
+  }
+
+  @Test
+  public void GetTrendsTest() throws IOException {
+    List<TrendRSS> results = parser.readTrendsFromStream(new FileInputStream(new File("/home/acnwigwe/step19-2020/project/src/test/java/com/google/sps/RSSFeed.xml"))).getTrends();
+    Assert.assertEquals(results.size(), 20);
+  }
+
```
Comment (niuamy) [inline project/src/test/java/com/google/sps/TrendRSSFeedTest.java:54]: is there a reason for this gap?

File: project/src/test/java/com/google/sps/TrendRSSFeedTest.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  @Test
+  public void CheckFreqAfterInvalidMakeTrendTest() {
+    Entity trend = service.makeTrend("Marsha P. Johnson", "hi");
+    Assert.assertEquals(0, trend.getProperty("traffic"));
+  }
+}
+
+
```
Comment (niuamy) [inline project/src/test/java/com/google/sps/TrendRSSFeedTest.java:92]: nit: remove space

File: project/src/test/java/com/google/sps/TrendRSSFeedTest.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  private final LocalServiceTestHelper helper = new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @Test
+  public void ReadTrendsTest() throws IOException {
+    TrendRSSFeed testfeed = parser.readTrendsFromStream(new FileInputStream(new File("/home/acnwigwe/step19-2020/project/src/test/java/com/google/sps/RSSFeed.xml")));
```
Comment (ashley3) [inline project/src/test/java/com/google/sps/TrendRSSFeedTest.java:45]: I think we should be able to reference this file relatively so without the "/home/acnwigwe/step19-2020/"

File: project/src/test/java/com/google/sps/TrendRSSFeedTest.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    Assert.assertNotNull(testfeed);
+  }
+
+  @Test
+  public void GetTrendsTest() throws IOException {
+    List<TrendRSS> results = parser.readTrendsFromStream(new FileInputStream(new File("/home/acnwigwe/step19-2020/project/src/test/java/com/google/sps/RSSFeed.xml"))).getTrends();
```
Comment (ashley3) [inline project/src/test/java/com/google/sps/TrendRSSFeedTest.java:51]: same here

**Pull request merged**